docker/nautobot_config.append.py

```diff
@@ -0,0 +1,33 @@
+LOG_LEVEL = "DEBUG" if DEBUG else "INFO"
```

Comment:
Do we really need this file or can we just make `development/nautobot_config.py` correct from the get-go?

Reply:
I attempted to do this with https://github.com/nautobot/nautobot/pull/199 my goal is to NOT have a custom nautobot config for the docker image and ultimately use the one from `nautobot-server init`